pyfolio/plotting.py

```diff
@@ -1660,3 +1662,126 @@ def plot_prob_profit_trade(round_trips, ax=None):
            xlim=(lower_plot, upper_plot), ylim=(0, y.max() + 1.))
 
     return ax
+
+
+def plot_cone(aggregate_returns, bounds, ax,
+              cone_std=(1, 1.5, 2.),
+              color='green',):
+    """
+    Plots the upper and lower bounds of an n standard deviation
+    cone of forecasted cumulative returns.
+
+    Parameters
+    ----------
+    aggregate_returns : pandas.core.frame.DataFrame
+        Cumulative out-of-sample returns.
+    bounds : pandas.core.frame.DataFrame
+        Contains upper and lower cone boundaries. Column names are
+        strings corresponding to the number of standard devations
+        above (positive) or below (negative) the projected mean
+        cumulative returns.
+    ax : matplotlib.Axes, optional
+        Axes upon which to plot.
+    cone_std : int, float, or list of int/float
+        Number of standard devations to use in the boundaries of
+        the cone. If multiple values are passed, cone bounds will
+        be generated for each value.
+    color : str, optional
+        Any matplotlib color
+
+    Returns
+    -------
+    ax : matplotlib.Axes
+        The axes that were plotted on.
+    """
+    for std in cone_std:
+        ax.fill_between(aggregate_returns.index,
+                        bounds[float(std)].iloc[:len(aggregate_returns)],
+                        bounds[float(-std)].iloc[:len(aggregate_returns)],
+                        color=color, alpha=0.5)
+    return ax
+
+
+def plot_multistrike_cones(is_returns, oos_returns, num_samples=1000,
```

Comment:
As it currently stands, this function can plot up to 3 strikes, depending on how the out-of-sample returns behave - we should either update the function name and/or docstring to make it clear that it's a fixed number of strikes, or allow the user to pass in a parameter to determine how many strikes they care about. OK with either, maybe the former is better in terms of limiting scope?